siriuspy/siriuspy/fofb/main.py

```diff
@@ -1001,6 +965,40 @@ def cmd_fofbctrl_dsblsysid(self, _):
         self._update_log('...done!')
         return True
 
+    # --- PS configuration matrix ---
+    def set_psconfig_mat(self, value):
+        """Set PSconfig matrix."""
+        self._update_log('Setting New PSConfigMat...')
+
+        # check size
+        mat = _np.array(value, dtype=float)
+        matb = mat
+        if mat.size != self._const.psconfig_size:
+            self._update_log(
+                f'ERR: Wrong PSConfig Size ({mat.size}, '
+                f'expected {self._const.psconfig_size}).')
+            return False
+
+        # reshape
+        nr_rows = self._const.nr_chcv
+        mat = _np.reshape(mat, [nr_rows, -1])
+
+        # select attributes
+        self._psconfig_mat = mat
+        self._corr_currloop_kp = self._psconfig_mat[:, 0]
+        self._corr_currloop_ti = self._psconfig_mat[:, 1]
```

Comment:
Could we use `ki` instead of `ti` for the new code?

Reply:
Yes, we are going to implement this in another deploy.

Reply:
To be implemented in a future PR